docs/source/users/theories.rst

```diff
@@ -0,0 +1,86 @@
+.. _theories_user:
+
+
+Scattering Theories in Holopy
+=============================
+
+The HoloPy :class:`.ScatteringTheory` classes know how to calculate scattered fields from detector and scatterer information. Each scattering theory is only able to work with certain specific scatterers.
+
+There are two broad classes of scattering theories in HoloPy: the
+:ref:`Lens-free<_lens_free>` theories which treat the recorded fields as the
+magnified image of the fields at the focal plane, and the
+:ref:`Lens<_with_lens>` theories which use a more detailed description of the
+effects of the objective lens. The lens-free theories usually do not need any
+additional parameters specified, whereas the lens theories need the lens's
+acceptance angle, which can be specified as either a fixed number of a
+:class:`.Prior` object, representing an unknown value to be determined in an
+inference calculation.
+
+All scattering theories in holopy inherit from the :ref:`ScatteringTheory` class.
+
+
+ScatteringTheory Methods
+------------------------
+
+HoloPy Scattering theories calculate the scattered fields through one of the following methods.
+
+    - :meth:`~.ScatteringTheory._raw_fields`
+      Calculates the scattering fields.
+    - :meth:`~.ScatteringTheory._raw_scat_matrs`
+      Calculates the scattering matrices.
+    - :meth:`~.ScatteringTheory._raw_cross_sections`
+      Calculates the cross sections.
+    - :meth:`~.ScatteringTheory._can_handle`
+      Checks if the theory is compatible with a given scatterer.
+
+If a theory is asked for the raw fields, but does not have a `_raw_fields` method, the scattering theory attempts to calculate them via the scattering matrices, as called by `_raw_scat_matrs`. More than one of these methods may be implemented for performance reasons.
+
+Be advised that the :class:`.ScatteringTheory` class is under active development, and these method names may change.
+
+
+.. _lens_free:
+
+
+Lens-Free Scattering Theories
+-----------------------------
+
+- :class:`.DDA`
+    * Can handle every :class:`.Scatterer` object in HoloPy
+    * Computes scattered fields using the discrete dipole approximation, as
+      implemented by ADDA.
+    * Requires the DDA package to be installed separately.
+- :class:`.Mie`
+    * Can handle :class:`.Sphere` objects, or :class:`.Spheres` through
+      superposision.
+    * Computes scattered fields using Mie theory.
```

Comment:
Can also handle multi-layered Spheres which should be mentioned here.

Reply:
Fixed by d3bbdce